homeassistant/core.py

```diff
@@ -30,6 +30,7 @@
     SERVICE_HOMEASSISTANT_RESTART, SERVICE_HOMEASSISTANT_STOP, __version__)
 from homeassistant.exceptions import (
     HomeAssistantError, InvalidEntityFormatError, ShuttingDown)
+from homeassistant.helpers.aiohttp_client import async_cleanup_websession
```

Comment:
Maybe you should rather import this in the function where you use it to keep `core` clean and without circular imports